hal_camera_default.te

```diff
@@ -18,6 +18,10 @@ allow hal_camera_default hal_power_default:unix_stream_socket connectto;
 allow hal_camera_default powerhal_socket:dir search;
 allow hal_camera_default powerhal_socket:sock_file write;
 
+allow hal_camera_default cashsvr:unix_stream_socket connectto;
+allow hal_camera_default cashsvr_socket:dir search;
+allow hal_camera_default cashsvr_socket:sock_file write;
```

Comment:
Use unix_socket_connect() macro.

Reply:
Perhaps everything else using sockets has to be fixed as well, not only my addition.
In this case, I'd rather say that it's better to just merge it as it is and then do a commit that does the cosmetic change that you are requesting on this plus all other non-CASH-related occurrences.